evaluation/create_api_usecase.md

```diff
@@ -0,0 +1,90 @@
+## Create An API
+### Primary Actor: Advanced User
+
+### Trigger
+The user wants to use a specific, not included API to solve a given task using a given function.
+
+### Preconditions
+1. There is at least one (built-in or previously created via the Developer Suite) function available for the user's account.
+
+### Main success scenarion
+1. The user clicks on "Switch to Developer Suite".
+2. The user confirms the warning pop-up to enter the advanced section Developer Suite.
+3. The user switches to the second tab "Manage APIs".
+4. The user switches to the second sub-tab "Create a new API".
+5. The user unlocks the protection mode to enter the edit mode.
+6. The user specifies general information about the API, information about the connection and additional parameter, required for the API to work.
+7. The user saves and finishes the API creation process.
+
+### Evaluation
+#### Task
+You, as an advanced user, want to add your favourite finance API for receiving current stock prices: **Yahoo Finance** to your account.
+Create a new API based on the following information:
+
+Map the new API to the function `stock_price_by_name`. Name it `My Evaluation Yahoo Finance API`. Set the priority to `High`. We skip the placeholder option for now. But do not forget to activate the API via the checkbox. The URL which belongs to the API is `https://apidojo-yahoo-finance-v1.p.rapidapi.com/stock/get-detail`. As a header, you need to specify an [RapidAPI](https://rapidapi.com/) key and value pair. If you have an account, use the provided credentials. Else, you can use this as the required header data:
```

Comment:
Habe die Funktionen stock_price_by_name und stock_price_by_symbol zusammengefasst in eine Funktion stock_price mit zwei Parametern (Company_symbol und Company_name) zusammengefasst. Die Evaluationsaufgabe funktioniert aber auch mit der neuen Funktion stock_price. Müsste man nur überall `stock_price_by_name` durch `stock_price` ersetzen.

Reply:
Falls es die favorisierte API ist, sollte man eher fordern, dass Prio 3 (preferred) ausgewählt werden soll.